rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+
+    @person = person
+    person.rental << self
+
+    @book = book
+    book.rental << self
```

Comment:
The rental called on the person and/or book instance is a collection/array, so it should be pluralized.
```ruby
    person.rentals << self

    @book = book
    book.rentals << self
```